packages/@n8n/nodes-langchain/nodes/chains/TextClassifier/TextClassifier.node.ts

```diff
@@ -151,6 +151,13 @@ export class TextClassifier implements INodeType {
 							rows: 6,
 						},
 					},
+					{
+						displayName: 'Enable Auto-Fixing',
+						name: 'enableAutoFixing',
+						type: 'boolean',
+						default: true,
+						description: 'Whether to enable auto-fixing for the output parser',
```

Comment:
Is this enough for people to understand that we will call the original model again if there's a parsing error?